core/templates/pages/about-page/about-page.component.spec.ts

```diff
@@ -16,161 +16,208 @@
  * @fileoverview Unit tests for the about page.
  */
 
+import { ComponentFixture, fakeAsync, TestBed} from '@angular/core/testing';
+import { EventEmitter, NO_ERRORS_SCHEMA, Pipe, PipeTransform }
+  from '@angular/core';
+
+import { AboutPageComponent } from './about-page.component';
```

Comment:
@bansalnitish, Can you please check?

Reply:
For the relative import here, we can say that since a component comprises four files (HTML, CSS, spec, and ts). So when importing stuff within the component we can use relative import.